projects/experimental/components/textfield/textfield.template.html

```diff
@@ -1,6 +1,5 @@
 <ng-container *ngIf="directive?.change$ | async"></ng-container>
 <ng-content select="input"></ng-content>
-<ng-content select="select"></ng-content>
```

Comment:
Is it expected deletion? this was necessary for native select dropdown on mobile

Reply:
Yes, I would refactor the selects separately, I have a plan )